packages/nimbuild-corejs/lib/module-resolver.js

```diff
@@ -133,10 +151,14 @@ function getModules({features, logger, overrideTargetPlatform, uaString}) {
         modules.corejs = results.modules;
         targetPlatform = results.targetPlatform; // need to re-assign in case target platofrm fails in `getCoreJSModulesByPlatform()`
     }
-    if (!detectIfBrowserHasFetch(targetPlatform)) {
+    if (!detectIfBrowserHasFeature(targetPlatform, 'fetch')) {
         modules.normal.push('whatwg-fetch');
     }
 
+    if (!detectIfBrowserHasFeature(targetPlatform, 'intersectionobserver')) {
```

Comment:
@neenhouse `caniuse` seems to return `false` here regardless if the browser is modern or not.

Reply:
Found the issue. [caniuse](https://github.com/Fyrd/caniuse/blob/master/features-json/intersectionobserver.json#L188-L205) is reporting that in Chrome 65 and up it has a [side note](https://github.com/Fyrd/caniuse/blob/master/features-json/intersectionobserver.json#L376). So when `caniuse-api` assert if the feature was supported it only checks for "y" and not "y #4".

Reply:
and now we wait https://github.com/Nyalab/caniuse-api/pull/90